plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py

```diff
@@ -318,7 +320,7 @@ def augmentation(
         model = get_gptq_peft_model(
             model,
             peft_config=peft_config,
-            auto_find_all_linears=peft_config.target_modules is None,
+            auto_find_all_linears=(peft_config.target_modules == PEFT_ALL_LINEAR),
```

Comment:
Perhaps we should check for both `["all-linear"]` and `"all-linear"`?

Reply:
Fair point @aluu317 

I see that @achew010 already tested with the common usage of the CLI, which would be

```
--target_modules all-linear
```
which seems to be parsed as a single string `"all-linear"`. 

We could guard against the case `["all-linear"]` though.